From 15b1dc72659e27f3610ae968f7ec90e441e62960 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Mon, 21 Nov 2011 23:08:18 +0800
Subject: [PATCH 176/256] ENGR00162690 vpu: Fix the issue of rotation 180
 degree in encoder

There may be mosaic if rotation is 180 degree in vpu encoder on mx5.
This patch fixed it, same solution is applied to mx6 platform.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 vpu/vpu_lib.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/vpu/vpu_lib.c b/vpu/vpu_lib.c
index c450dc2..715a150 100644
--- a/vpu/vpu_lib.c
+++ b/vpu/vpu_lib.c
@@ -1344,7 +1344,6 @@ RetCode vpu_EncStartOneFrame(EncHandle handle, EncParam * param)
 		vpu_mx6q_swreset(0);
 
 	if (pEncInfo->rotationEnable) {
-		rotMirEnable = 0x10;	/* Enable rotator */
 		switch (pEncInfo->rotationAngle) {
 		case 0:
 			rotMirMode |= 0x0;
@@ -1364,7 +1363,6 @@ RetCode vpu_EncStartOneFrame(EncHandle handle, EncParam * param)
 		}
 	}
 	if (pEncInfo->mirrorEnable) {
-		rotMirEnable = 0x10;	/* Enable mirror */
 		switch (pEncInfo->mirrorDirection) {
 		case MIRDIR_NONE:
 			rotMirMode |= 0x0;
-- 
1.8.0

